packages/react/src/component-tests/IcCard/IcCard.cy.tsx

```diff
@@ -185,11 +196,12 @@ describe("IcCard", () => {
     cy.get('[slot="image-mid"]').should(BE_VISIBLE);
     cy.get("ic-status-tag").should(BE_VISIBLE);
     cy.get("ic-button").should(BE_VISIBLE).should(HAVE_LENGTH, "2");
+
+    cy.checkA11yWithWait();
     cy.compareSnapshot({
-      name: "renderThreeCard",
-      testThreshold: DEFAULT_TEST_THRESHOLD + 0.05,
+      name: "render-three",
```

Comment:
Not sure this is the best name to describe the snapshot. Aware it explains the description of the test, but I'm not sure the description of the test is the best either because there's not three variants of card.